downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc

```diff
@@ -38,9 +38,13 @@ The automation hub UI is available by default at:
 https://<hub node>:444
 ----
 
-.Accessing Automation EDA UI
+Login as the admin user with the password you created for *controller_admin_password*.
+
+.Accessing Automation {EDAName} UI
```

Comment:
I'm not sure automation is needed here? I think the official product is simply Event-driven Ansible. If you want to include it, it should be lowercase since it's not part of the product name.